distributed-press/srv1.distributed.press/roles/install-api/tasks/install-api.yml

```diff
@@ -0,0 +1,89 @@
+---
+- name: Add nodejs repo key
+  apt_key:
+    url: https://deb.nodesource.com/gpgkey/nodesource.gpg.key
+    state: present
+
+- name: Add nodejs repo
+  apt_repository:
+    repo: "deb https://deb.nodesource.com/node_15.x buster main"
+    filename: nodesource
+    state: present
+
+- name: Install nodejs
+  apt:
+    update_cache: true
+    pkg:
+      - nodejs
+      - screen
+      - git
+
+- name: git clone https://github.com/hyphacoop/api.distributed.press.git
+  become_user: compost
+  git:
+    repo: 'https://github.com/hyphacoop/api.distributed.press.git'
+    dest: '/home/compost/api.distributed.press'
+    clone: yes
+    update: no
+
+- name: Create Data directory if doesn't exists
+  file:
+    path: /home/compost/api.distributed.press/data
+    owner: compost
+    group: users
+    state: directory
+    mode: '0700'
+
+- name: Check if API db.json exists
+  stat:
+    path: /home/compost/api.distributed.press/data/config.json
+  register: config_exists
+
+- name: Copy config.json /home/compost/api.distributed.press/
+  template:
+    src: config.json
```

Comment:
So this is from the encrypted blob?

Reply:
Yup it get it from https://github.com/hyphacoop/ansibles/blob/distributed-press/distributed-press/srv1.distributed.press/roles/install-api/templates/config.json